setup/setup.sh

```diff
@@ -1,25 +1,47 @@
 #!/usr/bin/env bash
 
-. ./env.sh
+if [ -z "$1" ]
+  then
+    echo "Please provide the OpenShift login Id."
+    echo "  e.g. ./setup/setup.sh developer"
+    echo ""
+    exit 1
+fi
+
+. ./env.sh $1 
 
 oc login -u $USER
 
-#oc delete project $CICD_PROJECT
-oc new-project $CICD_PROJECT 2> /dev/null
+
+oc delete project $DEV_PROJECT
+oc new-project $DEV_PROJECT 2> /dev/null
```

Comment:
Has this been tested? I would expect that if a project has content (e.g., running pods), it could take a long time for it to be a deleted and that happens asynchronously, meaning the command returns immediately and the subsequent new-project command would fail.